sequel/benchmarks/bm_sequel_scope_where.rb

```diff
@@ -0,0 +1,33 @@
+require 'bundler/setup'
+require 'sequel'
+require_relative 'support/benchmark_sequel'
+
+DB = Sequel.connect(ENV.fetch('DATABASE_URL'))
+
+DB.create_table(:users) do
+  primary_key :id
+  String :name, size: 255
+  String :email, size: 255
+  DateTime :created_at, null: true
+  DateTime :updated_at, null: true
+end
+
+DB.add_index :users, :email, unique: true
+
+class User < Sequel::Model
+  self.raise_on_save_failure = true
+end
+
+User.strict_param_setting = false
```

Comment:
On using setters we have to set all the fields separately and also primary key separately. If there are more number of parameters this could be a problem. So why not use this?